tests/models/test_rl/unit/test_agents.py

```diff
@@ -18,43 +18,44 @@ def setUp(self) -> None:
     def test_base_agent(self):
         agent = Agent(self.net)
         action = agent(self.state, 'cuda:0')
-        self.assertIsInstance(action, int)
+        self.assertIsInstance(action, list)
 
 
 class TestValueAgent(TestCase):
 
     def setUp(self) -> None:
         self.env = gym.make("CartPole-v0")
         self.net = Mock(return_value=torch.Tensor([[0.0, 100.0]]))
-        self.state = torch.tensor(self.env.reset())
-        self.device = self.state.device
+        self.state = [self.env.reset()]
+        self.device = torch.device('cpu')
         self.value_agent = ValueAgent(self.net, self.env.action_space.n)
 
     def test_value_agent(self):
 
         action = self.value_agent(self.state, self.device)
-        self.assertIsInstance(action, int)
+        self.assertIsInstance(action, list)
+        self.assertIsInstance(action[0], int)
 
     def test_value_agent_GET_ACTION(self):
```

Comment:
pls no capital in function name...

Reply:
No problem, i am used to having a main test name and then edge cases in caps. Ill change it